### PR TITLE
Update color-picker extension

### PR DESCRIPTION
**Pull request by xilopaint**
## Description

- Improved the `Organize Colors` command to dynamically fetch the frontmost application and display its icon in the paste action.

## Screenshot

color-picker-1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/color-picker/swift/color-picker/.gitignore
Comment (LitoMore) [inline extensions/color-picker/swift/color-picker/.gitignore:1]: This is unnecessary. Let's remove this file.

File: extensions/color-picker/src/utils.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { getPreferenceValues, Icon, Image, Keyboard, List } from "@raycast/api";
-import { HistoryColor, HistoryItem } from "./types";
 import ColorJS from "colorjs.io";
+import { HistoryColor, HistoryItem } from "./types";
```
Comment (LitoMore) [inline extensions/color-picker/src/utils.ts:3]: I saw so many unrelated changes on the PR. You should modify relevant code only.

Notes unrelated changes are not good for `git blame`.

Please revert all these unrelated changes. Thanks.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: extensions/color-picker/swift/color-picker/.gitignore
Comment (xilopaint) [reply extensions/color-picker/swift/color-picker/.gitignore:1]: I need some `.gitignore` rule. Without this file I see hundreds of files added in the source control. Also, the Raycast VS Code extension creates a new `.gitignore` file for the Swift directory after running the `Add Swift Support` command. So, it's not a strange approach for extensions with Swift support.

File: extensions/color-picker/src/utils.ts (hunk 1 above)
Comment (xilopaint) [reply extensions/color-picker/src/utils.ts:3]: This is result of VS Code `Organize Imports` command. It sorts imports and removes unused ones. I have seen a lot of extensions with unused imports which are basically dead code. I even think that this should be a formatting routine included in the CI.

**Comment by pernielsentikaer** [comment]
@samuelkraft, do you have an idea why it suddenly fails? It looks like you added some of the code that's now producing errors

File: extensions/color-picker/src/utils.ts (hunk 1 above)
Comment (LitoMore) [reply extensions/color-picker/src/utils.ts:3]: > By the way, you made the same change in your PR: [#13896 (files)](https://github.com/raycast/extensions/pull/13896/files)

I only changed those files in which I worked.

**Comment by xilopaint** [comment]
> @pernielsentikaer It's related to the `@raycast/eslint-config` dependency.

I reverted back to 1.0.8 in the last commit and the check still doesn't pass.

File: extensions/color-picker/src/utils.ts (hunk 1 above)
Comment (xilopaint) [reply extensions/color-picker/src/utils.ts:3]: Yes, but you didn't need to resort the imports to implement the feature your PR was intended for. It's the same here.

**Comment by xilopaint** [comment]
Thank you, @samuelkraft. It worked!

@pernielsentikaer the PR is ready.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥
